### DISCUSSION TITLE
no funciona en pixel 8 pro con Android 14. que debo hacer

### DISCUSSION THREAD

**Discussion by Carlos1988rm** [AndreyPavlenko/Fermata#308] [Q&A]
no funciona en pixel 8 pro con Android 14. que debo hacer ?

**Comment by divebombdom** [comment]
Having the same issue in my pixel 8 pro. Just trying to be patient.

**Reply by Carlos1988rm** [reply]
pues yo a parte de paciente, querría que al menos se supiese si se está trabajando en que Kinginstaller valga en Android 14

**Reply by divebombdom** [reply]
I agree. Not knowing is an uncomfortable position to be in.
